de_web_plugin.cpp

```diff
@@ -12668,7 +12668,7 @@ void DeRestPluginPrivate::processTasks()
             {
                 onAir++;
 
-                if (onAir >= maxOnAir)
```

Comment:
I'm concerned with these, are they really needed for the issue that the PR fixes?

Pushing too many commands in the lower level APS queue may cause more congestion and waiting times.
Since the device code doesn't use the tasks anymore (later superseded by StateChange mechanic) I'd prefer to keep the current limits here until we have a proper tracking system.

Reply:
Well, as described, then the limits are not correctly set or better to say it is irritating 😄 

Regardless, that is not really required for the fix, so I reverted it.

Reply:
Thanks :) I've tried to raise the limit in the past, causing weird issues in some networks.